### PR TITLE
Bench 258 unit tests for order serializer

### PR DESCRIPTION
**Pull request by NogaAD**

### CODE REVIEW

File: answerking_app/tests/test_unit/test_unit_order_serializer.py [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+
+    def test_order_serializer_id_field_content(self):
+        order: Order = Order.objects.first()
+        test_serializer_data: ReturnDict = OrderSerializer(order).data
+        expected: int = order.id
+        actual: int = test_serializer_data["id"]
+        self.assertEqual(actual, expected)
+
+    def test_order_serializer_products_details_content(self):
+        order: Order = Order.objects.first()
```
Comment (JoeCSykes) [inline answerking_app/tests/test_unit/test_unit_order_serializer.py]: would probably use objects.all() to get all orders and then test that only one order is returned then set order to orders[0]. See product serializer in dev for example. would do this method wherever you are doing objects.first(). this way we are confirming that we are getting the order we want.

File: answerking_app/tests/test_unit/test_unit_order_serializer.py [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+        test_serializer_data: ReturnDict = OrderSerializer(order).data[
+            "lineItems"
+        ][0]
```
Comment (JoeCSykes) [inline answerking_app/tests/test_unit/test_unit_order_serializer.py:86]: might also want to assert that there is only one object in lineitems

File: answerking_app/tests/test_unit/test_unit_order_serializer.py [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+            new_order_serializer_data["lineItems"],
+            list(new_order_object.lineitem_set.values()),
+        )
+
+    @freeze_time(frozen_time)
+    @mock.patch(
+        serializer_path + "OrderSerializer.products_check",
+        return_value=[],
+    )
+    def test_order_create_with_retired_product(self, products_check_mock):
```
Comment (JoeCSykes) [inline answerking_app/tests/test_unit/test_unit_order_serializer.py]: could you indicate on each test name whether the test should pass or fail (i.e. whether we are expecting a status in the 200 range or the 400 range)

Comment (NogaAD) [reply answerking_app/tests/test_unit/test_unit_order_serializer.py]: all of them should return 200. But I can create more tests, for non-valid data

File: answerking_app/tests/test_unit/test_unit_order_serializer.py (hunk 2 above)
Comment (NogaAD) [reply answerking_app/tests/test_unit/test_unit_order_serializer.py:86]: done

File: answerking_app/tests/test_unit/test_unit_order_serializer.py (hunk 3 above)
Comment (JoeCSykes) [reply answerking_app/tests/test_unit/test_unit_order_serializer.py]: ok well if thats the case put _pass at the end of all of them 👍

**Review by MichaelOSullivanAnswer — APPROVED** [review]

File: answerking_app/tests/test_unit/test_unit_order_serializer.py (hunk 3 above)
Comment (NogaAD) [reply answerking_app/tests/test_unit/test_unit_order_serializer.py]: done

**Review by JoeCSykes — APPROVED** [review]

**Pull request merged**